doc/contributing.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+This is for people who want to contribute to the project.
+
+## How to get started
+
+First of all, you need to fork the repository and clone it locally.
+After that, we've prepared a docker setup to help you get started quickly.
+
+Install docker and docker-compose on your machine, then run the following commands:
+
+```bash
+docker-compose up -d
```

Comment:
```suggestion
docker-compose up --detach
```

I prefer to use the long option, `--detach` is easier to understand than `-d`. But it can stay as is if you prefer.

Reply:
Done. Let me know if this is ok :)